devito/operator/profiling.py

```diff
@@ -230,10 +235,16 @@ def summary(self, args, dtype, reduce_over=None):
                 traffic = np.nan
 
             # Runtime itermaps/itershapes
-            itermaps = [OrderedDict([(k, int(subs_op_args(v, args)))
-                                     for k, v in i.items()])
-                        for i in data.itermaps]
-            itershapes = tuple(tuple(i.values()) for i in itermaps)
+            try:
+                itermaps = [OrderedDict([(k, int(subs_op_args(v, args)))
+                                         for k, v in i.items()])
+                            for i in data.itermaps]
+                itershapes = tuple(tuple(i.values()) for i in itermaps)
+            except TypeError:
+                # E.g., a section comprising just function calls
```

Comment:
redundant?

Reply:
I wouldn't say so?

Reply:
It is:
```
                # E.g., a section comprising just function calls    
                # E.g., a section comprising just function calls, or at least
                # a sequence of unrecognized or non-conventional expr statements
```
Why the part:
`# E.g., a section comprising just function calls`
is duplicated?